geo/src/algorithm/simplify_vw.rs

```diff
@@ -310,14 +350,18 @@ where
         }
         // if removal of this point causes a self-intersection, we also remove the previous point
         // that removal alters the geometry, removing the self-intersection
-        // HOWEVER if we're within 2 points of the absolute minimum, we can't remove this point or the next
+        // HOWEVER if we're within 1 point of the absolute minimum, we can't remove this point or the next
```

Comment:
Does this doc change reflect a bug in the existing documentation, or does this reflect a change in behavior from this PR?

Reply:
It's a change in the behaviour: https://github.com/georust/geo/pull/1052/commits/8e9aa6babe00863c8f62134d49e5dbc0c9751c91#diff-ba8c428d032bd24b196bd2649c0c7985dff438f05dfebd31814400adf9b39d71R583-R584